ush/templates/FV3LAM_wflow.xml

```diff
@@ -86,6 +87,10 @@ Directories and files.
 <!ENTITY OBSPATH                  "{{ obspath }}">
 <!ENTITY CYCLE_BASEDIR            "{{ cycle_basedir }}">
 <!ENTITY NWGES_BASEDIR            "{{ nwges_basedir }}">
+{%- if do_recenter %}
+<!ENTITY ENSCTRL_CYCLE_BASEDIR    "{{ ensctrl_cycle_basedir }}">
+{%- endif %}
```

Comment:
Line 90-92:  It should be fine to define the variable ENSCTRL_CYCLE_BASEDIR  even we don't do "recenter".